### PR TITLE
[CP] LRSD-5420 - Automate the manual steps into the quickstart CX Site initializer

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

**Comment by amosfong** [comment]
@abfranca did you see the confluence article? I think there is more stuff there. Or is this a partial PR first?

**Comment by abfranca** [comment]
@amosfong yeah, i've asked about that on our channel, since the confluence article mentions other tickets for other steps i thought there would be more tickets like this for the other missing parts. i'll be working on them now, thanks!

**Review by jennyhchen — CHANGES_REQUESTED** [review]

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-quickstart/client-extension.yaml [hunk 1]
```diff
@@ -1,13 +1,27 @@
 liferay-customer-site-initializer-quickstart:
     name: Liferay Customer Site Initializer Quickstart
-    oAuthApplicationHeadlessServer: liferay-customer-site-initializer-quickstart-oahs
+    oAuthApplicationHeadlessServer: liferay-customer-site-initializer-code-oahs
     siteExternalReferenceCode: LIFERAY_CUSTOMER
```
[the remark this answers is not in the document]
Comment (abfranca) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-quickstart/client-extension.yaml]: Done!

**Comment by abfranca** [comment]
ci:test:sf

**Comment by amosfong** [comment]
how come you removed a bunch of test accounts?

**Comment by abfranca** [comment]
@amosfong it was on the doc, should i add them back?

**Comment by amosfong** [comment]
oh...haha yea sorry that was unclear. These were originally steps for UAT and prod. Yea can you add them back, thanks!

**Comment by abfranca** [comment]
@amosfong they're back!

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-instance-settings/client-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Currently not working
```
Comment (amosfong) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-instance-settings/client-extension.yaml:1]: Can you remove these comments and put it in the ticket instead?

**Comment by ryanschuhler** [comment]
closing for https://github.com/liferay-customer-solutions/liferay-portal/pull/306 (also removed the instance settings cx for now)

**Pull request closed without merging**